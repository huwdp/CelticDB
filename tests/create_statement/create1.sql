CREATE TABLE test1;
CREATE TABLE test2;
SHOW TABLES;