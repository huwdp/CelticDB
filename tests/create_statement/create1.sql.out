Parser: Create table named 'test1'
Parser: Create table named 'test2'
Parser: Show tables
Command: Create statement
Command: Create statement
Command: Show tables statement
Table name: test1
	Row count: 0
	Column count: 0
Table name: test2
	Row count: 0
	Column count: 0
